### DISCUSSION TITLE
孤單對健康的危害等於每天多抽一根菸

### DISCUSSION THREAD

**Discussion by a3240281370** [1101-GS4719-C/Python-Community-GS4719-C#1408] [General]
我最近看到一個很有趣的研究，科學家在統整超過 70 份相關研究後發現，長期不與人接觸所造成的寂寞感，會增加約 30% 的死亡率
​單從這個角度看，孤單寂寞的危害程度約等於每天抽一支煙、喝 5 罐啤酒或久坐超過 8 個小時不運動
什麼！！！寂寞也是錯嗎！
撇開死亡風險，寂寞對健康的負面影響，還包括中風等心血管疾病風險增加 30%、失智風險增加 50% 等等。

哎不說了，不知道多抽幾根菸了，肺都黑了😞
​

**Comment by Jamesmoneya** [comment]
孤單的人靠抽菸解除孤單的感覺，好恐怖超級傷身體的奉勸大家不要抽菸，也趕快找到適合的人可以趕快再一起，跟喜歡的人再一起真的是世界上最開心的一件事

**Comment by Alexgns** [comment]
我認為找到自己熱愛並喜歡的事情是確保心靈不感到孤獨的重要因素。當我們投身於自己熱愛的事業或興趣時，我們可以充實自己的生活，並找到內在的滿足感。無論是追求個人愛好、深入學習某個領域，或是投身於藝術、音樂、運動等活動，都能讓我們在獨自一人時感到充實而不寂寞。

**Comment by alicia0725** [comment]
我認為孤單的標準不該是與人接觸與否，若能夠獨處時也感受到滿足不寂寞，應該也不能夠稱為孤單。

**Comment by ashleyy-huang** [comment]
對我來說孤單的感覺比較像是在環境中沒有自我認同的感覺，因為如果我在獨處時也會享受一個人的時光，但如果是在一群人中沒有和朋友交流的話那對我來說就是有些孤單了

**Comment by lyien1021** [comment]
那這樣我好奇如果是真正享受孤獨的人呢？說不定危害程度就不就不會像每天抽一支煙一樣？

**Comment by larry40305** [comment]
改善社交環境和心理健康可以為我們帶來更長遠的益處。不妨試著尋求更多與他人連結的方式，建立支持系統，並注意照顧自己的整體健康。

**Comment by youtinglin** [comment]
原來孤單對於人體的危害如此的大，或許因為人類本身就是群體動物。我也覺得有時候如果太少與他人互動，心情會很低落，之前有看到一篇報導說比較會社交的人壽命會較長也比較容易成功，可是並不代表比較內向就不好，內向的人還是可以與他人有保持良好的社交關係。

**Comment by MaxChi870** [comment]
我認為研究所說的孤獨應是屬於當人們渴望與群體互動時，內心所產生的寂寞感，但若是一個人能自我娛樂，或專注於自我的興趣中，而遠離人群，就算外界看來是孤單的，當事人的內心也可能比一般人豐富。

**Comment by zhiron544** [comment]
我意識到寂寞不僅是一種心理狀態，還是一種生理危機。我覺得現代社會的人際關係越來越淡薄，需要更多的關懷和溝通。我想知道如何減少寂寞感，並提高生活品質。
